docs/protocol.md

```diff
@@ -590,6 +591,10 @@ Value is string "host:port".
 
 Send claim message to `host:port` when response is being sent.
 
+### Transport Header `cn` -- Caller Name
+
+Value is name of the service making the call.
```

Comment:
This should correlate with the calls to register that the service is making. You think that's enough to lookup all desired info? My thought is that the service name is the key into this type of data.